File: test_aws.py
from workload import Workload
from platform_adapter.aws.aws import AWS

TASKS = 5
workload_path = "workload_with_top_mods.json"
wl = Workload(AWS(), workload_path)

wl.play(
    tasks=TASKS,
    options={"workload_path": workload_path}
)

File: test/test_aws.go
package main

import (
	"rb"
	"rb/platform_adapter"
	"rb/platform_adapter/aws"
	"fmt"
)

func main() {
	var a = aws.AWSAdapter{
		Workload_path: "../files/workloads_5.json",
	}
	var m platform_adapter.PlatformAdapter = &a
	
	rb, err := rb.NewReqBench(m, "../files/workloads_5.json")
	if err != nil {
		panic(err)
	}

	err = rb.StartWorker(nil) //start 
	err = rb.DeployFuncs()
	tasks, timeout, totalTime := 1, 2, 1
	rb.Play(tasks, timeout, totalTime, nil)
	//rb.KillWorker(nil)
	//a.DeleteAll()
	
	if err != nil {
		fmt.Println(err)
	}
}
